packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -202,6 +219,17 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
     _.invoke(props, 'onDateChange', e, { ...props, value: targetDay });
   };
 
+  const calendarElement = createShorthand(DatepickerCalendar, calendar, {
+    defaultProps: () =>
+      getA11yProps('calendar', {
+        className: datepickerCalendarClassName,
+      }),
+    overrideProps: () => ({
+      ...calendarOptions,
+      onDateChange: handleChange,
+    }),
```

Comment:
```suggestion
    overrideProps: (predefinedProps) => ({
      ...calendarOptions,
      onDateChange: handleChange(predefinedProps),
    }),
```

It should also call `predefinedProps.onDateChange`. Otherwise if a user will pass `onDateChange` to `calendar` shorthand it will be never called.